src/kernel.jl

```diff
@@ -509,6 +509,15 @@ moffat(α::Real, β::Real)                 = moffat(α, β, ceil(Int, (α*2*sqrt
     sum(x.^2)
 end
 
+function butterworth(n::Real,Wn::Real,ls::Tuple{Integer, Integer})
```

Comment:
The style should be consistent, you can additionally check out https://github.com/invenia/BlueStyle (but try to keep the same to the current codebase you're working on)

```suggestion
function butterworth(n::Real, Wn::Real, ls::Tuple{Integer,Integer})
```